acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-vendor-address-included-with-batch-voucher.feature

```diff
@@ -106,11 +108,11 @@ Feature: Check vendor address included with batch voucher
     When method POST
     Then status 201
     * def batchVoucherExportId = $.id
-    * call pause 1000
 
     # ============= get export later to give it time to create the batch voucher ===================
     Given path 'batch-voucher/batch-voucher-exports', batchVoucherExportId
     And headers headersUser
+    And retry until response.status == 'Error'
```

Comment:
Should it be Completed status here?

Reply:
I thought the same but I checked on the snapshot environment and got the error status